tools/test/h5dump/CMakeTests.cmake

```diff
@@ -1567,5 +1567,5 @@ if (HDF5_ENABLE_ROS3_VFD_DOCKER_PROXY)
   ADD_H5_S3TEST (tattrintsize 0 anon s3 h5dumpros3 --filedriver=ros3 --vfd-name=ros3 --s3-cred=\(,,\) --endpoint-url=http://localhost:${h5dump_s3tests_port})
   ADD_H5_S3TEST (tattrintsize 0 profile http localhost:${h5dump_s3tests_port}/h5dumpros3 --vfd-name=ros3)
   ADD_H5_S3TEST (tattrintsize 0 profile s3 h5dumpros3 --filedriver=ros3 --vfd-name=ros3 --endpoint-url=http://localhost:${h5dump_s3tests_port})
+  ADD_H5_S3TEST (tattrintsize 0 foo s3 h5dumpros3 --endpoint-url=http://localhost:${h5dump_s3tests_port})
```

Comment:
Please change "foo" to something more applicable - like "filename" as this will show up in the test results.

Reply:
Done.